src/components/Result.tsx

```diff
@@ -77,5 +91,3 @@ const Result = () => {
     </div>
   );
 };
-
-export default Result;
```

Comment:
This doesn't look right - did you remove the export accidentally?

Reply:
Yep, definitely accidentally.